service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseManagementOrderCaseLinkNotificationHandler.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.fpl.handlers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Representative;
+import uk.gov.hmcts.reform.fpl.model.event.EventData;
+import uk.gov.hmcts.reform.fpl.service.InboxLookupService;
+import uk.gov.hmcts.reform.fpl.service.RepresentativeService;
+import uk.gov.hmcts.reform.fpl.service.email.NotificationService;
+import uk.gov.hmcts.reform.fpl.service.email.content.CaseManagementOrderEmailContentProvider;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.apache.commons.lang.StringUtils.isNotBlank;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.DIGITAL_SERVICE;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class CaseManagementOrderCaseLinkNotificationHandler {
+    private final ObjectMapper objectMapper;
+    private final InboxLookupService inboxLookupService;
+    private final NotificationService notificationService;
+    private final RepresentativeService representativeService;
+    private final LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration;
+    private final CaseManagementOrderEmailContentProvider caseManagementOrderEmailContentProvider;
+
+    public void sendCMOCaseLinkNotifications(final EventData eventData) {
```

Comment:
Nothing major but there is quite a bit of repetition going on with this name and the others, wonder if we can simplify it slightly for some nicer reading.
Currently you have `caseManagementOrderCaseLinkNotificationHandler.sendCMOCaseLinkNotifications` which is a bit of a mouthful and repetitive. I think the method name could be just `sendNotifications` as with the variable name the way it is it is clear what it is doing, you have a CaseLinkNotificationHandler and you are sending the notifications.

The other method names in this class could be simplified as well due to the current context we are in by being in the class, maybe `sendToLocalAuthority` and `sendToRepresentatives`?

Reply:
It is the same for the other methods in the other classes as well. Whilst verbosity can be useful I think it is also useful to remember that it should read nicely in the context of their own classes.